server/scopeserver/oidc.py

```diff
@@ -0,0 +1,161 @@
+" Abstractions over OpenID Connect APIs "
```

Comment:
Maybe some links to docs would help here too

Reply:
Added to README